controllers/login.js

```diff
@@ -0,0 +1,25 @@
+const ModelUser = require('../models/user')
+const Guid = require('guid')
+
+module.exports.login = async function login (req, res) {
+  try {
+    const { email, password } = req.body
+    const User = await ModelUser.findOne({email: email})
```

Comment:
I think you can use  await ModelUser.findOne({ email })

Reply:
Solved